FoundationBuilder/include/MetadataOutput.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <string>
+
+namespace geom_utils {
+	class Mesh;
+	void writeMetadata(const Mesh& inputModel, const Mesh& foundation, const std::string& metadataFile, const std::string& inputModelName, const std::string& foundationName, const bool& buildIn, const float& timeOfConstructing, const float& foundationMinimalZCoord);
```

Comment:
Please prefer passing simple types like bool, float, etc. by value.
Passing by const ref makes sense if argument is non-trivial type with non-trivial copy ctor, like our Mesh type. It has 'heavy' copy ctor, so we have some profit when passing it by const ref without copying. The same cant be said about simple types.